### PR TITLE
Documentation and code clarification.

### PR DESCRIPTION
**Pull request by feerrenrut**
Some suggestions for PR https://github.com/nvaccess/nvda/pull/8853

### CODE REVIEW

**Comment by feerrenrut** [comment]
I'd really like to get this merged nvaccess#8853 merged in. But first I would like either @bramd or @leonardder to take a look. Please could one of you take a look at this?

File: source/brailleDisplayDrivers/freedomScientific.py [hunk 1]
```diff
@@ -37,6 +38,9 @@
 }
 
 #: Number of cells of Focus first generation displays
+#  The assumption is that any displays with the following cell counts are due to three cells at the
+#  beginning/end of the display are used as status cells, and an extra blank cell to separate status
+#  from normal cells. These devices require a special translation table: L{FOCUS_1_TRANSLATION_TABLE}
```
Comment (LeonarddeR) [inline source/brailleDisplayDrivers/freedomScientific.py:43]:
```suggestion
#  from normal cells. These devices require a special translation table: L{FOCUS_1_TRANSLATION_TABLE}
#  This line of displays is known as the first generation Focus displays.
```

**Review by LeonarddeR — APPROVED** [review]
I would personally advice to have @bramd or someone else with a display have one additional round of testing.

**Comment by bramd** [comment]
@feerrenrut Thanks for your work, looks good to me. I'll give this one more round of testing with the display I have.

**Comment by bramd** [comment]
Everything still works as expected with the Focus 40 blue.

**Pull request merged**